src/Standards/Generic/Tests/ControlStructures/ControlStructuresBracketsUnitTest.inc

```diff
@@ -0,0 +1,170 @@
+<?php
+
+$a = true;
+
+// Warning
```

Comment:
Test not needed (as per previous comment).

Reply:
Removed